vue/components/ui/atoms/button-icon/button-icon.vue

```diff
@@ -269,7 +273,7 @@ export const ButtonIcon = {
                             : this.paddingBase
                         : this.paddingRight
                 }px`,
-                "border-radius": `${this.size}px`
+                "border-radius": `${this.borderRadius || this.size}px`

```

Comment:
This won't work for `borderRadius = 0` right? Use a comparison with `null` or something